apps/nrm/views/does-pv-need-support.html

```diff
@@ -0,0 +1,17 @@
+{{<partials-page}}
+  {{$page-content}}
+    <p>{{#t}}pages.does-pv-need-support.paragraph-1{{/t}}</p>
+    <ul class="govuk-list govuk-list--bullet">
+        <li>{{#t}}pages.does-pv-need-support.bullet-1{{/t}}</li>
+        <li>{{#t}}pages.does-pv-need-support.bullet-2{{/t}}</li>
+        <li>{{#t}}pages.does-pv-need-support.bullet-3{{/t}}</li>
+        <li>{{#t}}pages.does-pv-need-support.bullet-4{{/t}}</li>
+        <li>{{#t}}pages.does-pv-need-support.bullet-5{{/t}}</li>
+    </ul>
+    <p><a href="#">{{#t}}pages.does-pv-need-support.link-text-1{{/t}}</a> {{#t}}pages.does-pv-need-support.link-text-2{{/t}}</p>
```

Comment:
Can the above be easily put in a Markdown? I won't hold up the PR for that

Reply:
@sulthan-ahmed would you prefer I resolve this type of issue in a separate PR or within it?

Reply:
Actioned in PR regarding MS-735 Do they need support

Reply:
np